lib/src/v3/models/views.dart

```diff
@@ -65,7 +65,6 @@ class SiteView with _$SiteView {
   @JsonSerializable(fieldRename: FieldRename.snake)
   const factory SiteView({
     required Site site,
-    required PersonSafe creator,
```

Comment:
This was not removed: https://github.com/LemmyNet/lemmy-js-client/blob/af760304d8d10203daf5cc2c17db3831053d1ae7/src/interfaces/views.ts#L58

Reply:
Actually... It was removed, but not updated in the TS api client! Nice catch! https://github.com/LemmyNet/lemmy/blob/dd5835fb6ee210af78ba876bf3e9f37ea1db521b/crates/db_views/src/site_view.rs#L10

Will send a PR to the TS repo

Reply:
Done: https://github.com/LemmyNet/lemmy-js-client/pull/49